_data/file_formats/itm_v1/feature_block.yml

```diff
@@ -1,17 +1,24 @@
 - desc: Opcode Number
   type: word
 - desc: |
-    Target type
-    - 0 = None
-    - 1 = Self (pre-projectile)
-    - 2 = Pre-target
-    - 3 = Party
-    - 4 = Everyone (inc. party)
-    - 5 = Everyone (excl. party)
-    - 6 = Everyone matching specific value of caster (or Party if cast by party member)
-    - 7 = Everyone matching specific value of target (In ITM: ranged ability type only, otherwise no target)
-    - 8 = Everyone (excl. caster)
-    - 9 = Self (post-projectile)
+    <b><a name="V1_Feature_Block_0x2">Target type</a></b>
+
+    - 0 ⟶ None
+    - 1 ⟶ <b><a name="V1_Feature_Block_0x2_1">Self</a></b>
+    - 2 ⟶ <b><a name="V1_Feature_Block_0x2_2">Projectile target</a></b>
+      - SPL global (casting) effects will not work with this type.
+      - Affects each target of the specified projectile, as it impacts them.
+    - 3 ⟶ <b><a name="V1_Feature_Block_0x2_3">Party</a></b>
+    - 4 ⟶ <b><a name="V1_Feature_Block_0x2_4">Everyone</a></b>
+    - 5 ⟶ <b><a name="V1_Feature_Block_0x2_5">Everyone except party</a></b>
+    - 6 ⟶ <b><a name="V1_Feature_Block_0x2_6">Caster group</a></b>
+    - 7 ⟶ <b><a name="V1_Feature_Block_0x2_7">Target group</a></b>
+      - Affects all creatures in the current area with the same <code><a href="../../files/ids/bgee/specific.htm">SPECIFIC.IDS</a></code> value as the ability target. (Ability must target a creature, not a location, container, door, etc...)
```

Comment:
This applies to the previous item as well, just that it takes the value from a different actor.

Reply:
> This applies to the previous item as well, just that it takes the value from a different actor.

I agree.